src/utils.py

```
@@ -38,9 +38,16 @@ def ensure(assertion, msg, *args):
     if not assertion:
         raise AssertionError(msg % args)
 
+def writable_dir(path):
```

good check, failing fast